### PR TITLE
Wsegvalv2

### PR DESCRIPTION
**Pull request by tklov11**
Oppdatert med støtte for WSEGVALV-nøkkelordet, tester og docs.

### CODE REVIEW

File: ecl2df/compdat.py [hunk 1]
```diff
@@ -134,6 +135,12 @@ def deck2dfs(deck, start_date=None, unroll=True):
                 rec_data["DATE"] = date
                 rec_data["KEYWORD_IDX"] = idx
                 wsegaicdrecords.append(rec_data)
+        elif kword.name == "WSEGVALV":
+            for rec in kword:  # Loop over the lines inside WSEGAICD record
```
Comment (berland) [inline ecl2df/compdat.py:139]: Comment here looks wrong..

Comment (tklov11) [reply ecl2df/compdat.py:139]: yes, it does. I'll fix.

**Comment by tklov11** [comment]
Do you have recommendations on other keywords that are tested vigorously, that I may have a look at?

**Review by berland — APPROVED** [review]
LGTM!

**Pull request merged**